mapproxy/request/base.py

```diff
@@ -279,7 +279,7 @@ def server_script_url(self):
         return self.script_url.replace(
             self.host_url.rstrip('/'),
             self.server_url.rstrip('/')
-        )
+        )[:-len(self.environ.get('SCRIPT_NAME', ''))]
```

Comment:
When behind a reverse proxy, this gets rid of `SCRIPT_NAME`, which doesn't exist on the backend.